### DISCUSSION TITLE
Will there be a option for lower quality download

### DISCUSSION THREAD

**Discussion by Abish4i** [audiamus/BookLibConnect#20] [Q&A] [answered]
I get this bigger files are better but I usually download in one go so I'd like to know will in near future be any medium quality size possible for download.

And this is really a very great program. 🙇 
Since last 2 months trying to decrypt 4byte activation key with no success.
Any ways just want to know will there be any function or option for files with lesser size.

**Comment by audiamus** [comment]
I have put this on the wish list.

At the moment the client is always requesting maximum audio quality. For newer books that means 44.1kHz, 128kb/s, for older books 22.05 kHz, 64 kb/s. In any case, what you get is up to the server.

Allowing more options for the quality request argument will have some implications on possible re-downloads. Currently, when you select a book for download, the program will check whether it has been downloaded before and if so, whether the downloaded file still exists. In that case, the book will not be downloaded again, to save time. With a variable quality option, the program has to know which setting was applied the first time, and if the new setting is higher, it should re-download.

For the time being, you are aware that you can download longer books as multiple parts? This may be helpful if you have a somewhat unreliable internet connection. And you are also aware that you can always reduce the audio quality with AAX Audio Converter, to get smaller files?

**Comment by audiamus** [comment, marked as answer]
>  I'd like to know will in near future be any medium quality size possible for download.

Implemented in [Version 0.5](https://github.com/audiamus/BookLibConnect/releases/tag/v0.5).

**Comment by Abish4i** [comment]
**Thanks for implementing this feature.** 
I've been waiting to download files with length between _30hrs_ to _75hrs_.

Thanks again for being a life saver
